language/src/regent/optimize_index_launches.t

```diff
@@ -472,6 +474,9 @@ local function analyze_noninterference_self(
     if analyze_index_noninterference_self(index, cx, loop_vars)
     then
       return true
```

Comment:
If we're going to make this function return two values, should we return a second one here? (Certain things can go rather strangely when unpacking `nil` values, so `false` would be a bit safer.)

Reply:
Fixed.